2d/camera-zoom/Actors/ZoomingCamera2D.gd

```diff
@@ -26,7 +29,7 @@ func _unhandled_input(event):
 
 func _set_zoom_level(value: float) -> void:
 	_zoom_level = clamp(value, min_zoom, max_zoom)
-	EVENTS.emit_signal("zoom_level_changed", _zoom_level)
+	self.emit_signal("zoom_level_changed", _zoom_level)
```

Comment:
Don't use self when calling methods on the instance, it's unnecessary in GDScript